kitchen.yml

```diff
@@ -27,3 +27,7 @@ suites:
   - name: ldap
     run_list:
       - recipe[test::ldap]
+
+  - name: plugins
```

Comment:
in `.circleci/config.yml` could you add the following so the new tests run:
```
      - kitchen/dokken-single:
          name: plugins
          suite: plugins
          requires:
            *lint_and_unit
``` 
Thanks!